assets/src/block-templates/register.js

```diff
@@ -22,10 +22,10 @@ export const registerBlockTemplates = ( blockTemplates ) => {
   const templates = blockTemplates || templateList;
   const postType = getCurrentPostType();
 
-  templates.map( ( blockTemplate ) => {
-    let { metadata, template, templateLock = false } = blockTemplate;
+  templates.forEach( ( blockTemplate ) => {
```

Comment:
We can also consider to replace to `for` instead. I don't think we'd need to `return` a value.

Reply:
That is why I am using the forEach, since we are not returning anything.